app/models/requirement.rb

```diff
@@ -0,0 +1,57 @@
+class Requirement < ActiveRecord::Base
+  belongs_to :task
+  belongs_to :skill
+  #has_many :assignments
+
+  validates :task, presence: true
+  validates :skill, presence: true
+
+  validates :minimum_people, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :maximum_people, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :desired_people, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validate :valid_people_numbers
+
+  STATUS_CHOICES_ARRAY = ['Empty', 'Inadequate', 'Adequate', 'Satisfied', 'Full']
```

Comment:
See the `status` method (which relies on stubs for assignments).  We compare how many people we have assigned with the desired/required numbers and return the appropriate status.  As of now, we don't store any of this in the db. (If we think performance will be an issue, we can cache whatever we like, but then we'll have to take care when assignments are changed.)